src/main/java/com/statful/client/framework/springboot/processor/GenericProcessor.java

```diff
@@ -0,0 +1,15 @@
+package com.statful.client.framework.springboot.processor;
+
+import com.statful.client.framework.springboot.common.MetricType;
+import com.statful.client.framework.springboot.common.ProcessedMetric;
+
+public abstract class GenericProcessor {
```

Comment:
Since this class has only one static method, instead of abstract, it could be considered final and it's constructor declared private to avoid initialization.